styles.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+  background-color: white;
+  color: rgb(60, 122, 60);
+  padding: 0;
```

Comment:
Do `padding` and `background-color` really do anything for you here? Check all of your CSS properties to confirm that none of them are extraneous.